src/main/webapp/user_profile.html

```diff
@@ -115,5 +49,25 @@ <h1>CREATE A NEW TASK: </h1>
             </form>
         </div>
     </div>
+    <div id="editTaskModal">
+        <div id="editModal">
+            <span id="edit-close-button" onclick="closeEditModal()">&times;</span>
+            <form id="edit-task-form" action="/tasks/edit" method="POST">
+                <h1>EDIT THE CURRENT TASK: </h1>
+                <div>
+                    <label for="edit-content-input">Task Detail:</label>
+                    <br/>
+                </div>
+                <textarea name="edit-content-input" id="edit-content-input"></textarea>
+                <br/>
+                <label for="edit-point-input">Rewarding Points: </label>
+                <input type="number" id="edit-point-input" name="edit-input" min="0" max="200">
+                <textarea name="task-id" id="task-id-input"></textarea>
```

Comment:
This would be better as `<input type="hidden">`, which is meant for passing along invisible data in a form.

Reply:
Okay. I would replace the task id input `textarea` with a hidden input in the next commit.